internal/otelcollector/config/metric/gateway/config_builder.go

```diff
@@ -212,3 +212,8 @@ func isRuntimeInputEnabled(input telemetryv1alpha1.MetricPipelineInput) bool {
 func isIstioInputEnabled(input telemetryv1alpha1.MetricPipelineInput) bool {
 	return input.Istio != nil && input.Istio.Enabled
 }
+
+// isOtlpInputEnabled
```

Comment:
I think that this comment was probably unintentionally committed, right?

Reply:
True, will remove it!